pkgs/by-name/py/pyamlboot/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyamlboot";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "superna9999";
+    repo = "pyamlboot";
+    rev = "refs/tags/${version}";
+    hash = "sha256-vpWq8+0ZoTkfVyx+2BbXdULFwo/Ug4U1gWArXDfnzyk=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    pyusb
+    wheel
+    setuptools
+  ];
+
```

Comment:
Please add https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion , because it's python and we all don't know what happens next

Reply:
Done